src/libs/actions/Task.js

```diff
@@ -551,8 +551,9 @@ function getAssignee(details) {
  * */
 function getShareDestination(reportID, reports, personalDetails) {
     const report = lodashGet(reports, `report_${reportID}`, {});
+    const personalDetail = personalDetails[0] || {};
```

Comment:
I don't think we need this.  The `personalDetails` is an object, and end up the personal detail always `{}`.